doc/lua_api.txt

```diff
@@ -754,34 +807,34 @@ Examples of sound parameter tables:
 
     -- Play locationless on all clients
     {
-        gain = 1.0, -- default
-        fade = 0.0, -- default, change to a value > 0 to fade the sound in
-        pitch = 1.0, -- default
+        gain = 1.0,   -- default
+        fade = 0.0,   -- default, change to a value > 0 to fade the sound in
+        pitch = 1.0,  -- default
     }
     -- Play locationless to one player
     {
         to_player = name,
-        gain = 1.0, -- default
-        fade = 0.0, -- default, change to a value > 0 to fade the sound in
-        pitch = 1.0, -- default
+        gain = 1.0,   -- default
+        fade = 0.0,   -- default, change to a value > 0 to fade the sound in
+        pitch = 1.0,  -- default
     }
     -- Play locationless to one player, looped
     {
         to_player = name,
-        gain = 1.0, -- default
+        gain = 1.0,  -- default
```

Comment:
Totally fine to line-up, but why do you use one space more than necessary in these examples?

Reply:
I was going for 2 spaces separation from the code as suggested by the Lua style guide

Reply:
Alright, I'm wrong in this point -- didn't know it was set to two spaces.